pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+"What does this PR do?" This pull request created to get all form answers when
+the manger select the employee from the list to show result.And this happend by
+employee (id) joined with form (id) ,and questions (id) in the query.
+
+"Does this Fix an Issue?"
+ "Related #34"
+
+"How to test this PR"
+1-You should be able to access database on heroku by url.
+2-When you can access to database by url, implement db_build.sql by db_build.js
+to create all tables with it's relations. 
+3-Implement dummyData.sql by
+db_build.js. 
+4-Go to check the tables data from terminal by make some basics
+query. 
+5-Make getForm.js consist of function that implement join query between
+form,question,employee in queries file. 
+6-Make route in index.js in controller
+as "(router.get('/api/forminfo/:employeeID', getForms);)". 
```

Comment:
Hussien! He doesn`t need steps you worked on it here, its enough if you say "npm test"

Reply:
done!